packages/vhd-lib/merge.integ.js

```diff
@@ -16,350 +17,360 @@ const { checkFile, createRandomFile, convertFromRawToVhd } = require('./tests/ut
 let tempDir = null
 let handler
 let disposeHandler
-jest.setTimeout(60000)
 
-beforeEach(async () => {
-  tempDir = await pFromCallback(cb => tmp.dir(cb))
+async function exists(file) {
+  try {
+    await fs.access(file, fs.constants.F_OK)
 
-  const d = await getSyncedHandler({ url: `file://${tempDir}` })
-  handler = d.value
-  disposeHandler = d.dispose
-})
+    return true
+  } catch (err) {
+    if (err.code === 'ENOENT') return false
+    throw err
+  }
+}
```

Comment:
Why?

Reply:
Since ```fs.exists()``` is deprecated (https://nodejs.org/api/fs.html#fsexistspath-callback), I created a function named ```exists()``` that use ```fs.access()``` with some treatments to get the same output that ```fs.exists()```, which is a boolean.

Reply:
`fs.exists` is deprecated, not `fs-extra.exists` as far as I know.

Where did you see this deprecation? Why don't I see it on master?